test-suite/exonum-py-tests/suite/common.py

```diff
@@ -123,3 +134,27 @@ def wait_api_to_start(network: ExonumNetwork) -> None:
                 break
             except ConnectionError:
                 time.sleep(0.5)
+
+
+def generate_config(
+    network: ExonumNetwork,
+    deadline_height: int = 10000,
+    consensus: dict = None,
+    artifact_name: str = ARTIFACT_NAME,
+    instances: dict = {},
```

Comment:
This is a bad practice to use a mutable value as a default value of the argument. Take a look [here](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html).